Source/WebKit/NetworkProcess/storage/NetworkStorageManager.cpp

```diff
@@ -367,10 +370,13 @@ NetworkQuotaManager& NetworkStorageManager::quotaManager()
 
         uint64_t quota = std::numeric_limits<uint64_t>::max();
         if (m_totalQuotaRatio) {
+            std::optional<uint64_t> volumeCapacity;
             if (m_volumeCapacityOverride)
-                quota = m_totalQuotaRatio.value() * m_volumeCapacityOverride.value();
+                volumeCapacity = m_volumeCapacityOverride;
             else if (auto capacity = FileSystem::volumeCapacity(m_path))
-                quota = m_totalQuotaRatio.value() * capacity.value();
+                volumeCapacity = WTF::roundUpToMultipleOf(defaultVolumeCapacityUnit, *capacity);
```

Comment:
```suggestion
                volumeCapacity = WTF::roundUpToMultipleOf(defaultVolumeCapacityUnit, * capacity);
```

Reply:
hmm it seems uncommon to add a space between * and `std::optional<>` to get its contained value?

Reply:
Right, I misread it.